### ISSUE TITLE
CREATE VIEW problem

### ISSUE THREAD

**Issue by walter-weinmann** [K2InformaticsGmbH/sqlparse#24]
Instead of 

```
view_def -> CREATE VIEW table opt_column_commalist
view_def -> AS query_spec opt_with_check_option

```

it should probably be

`view_def -> CREATE VIEW table opt_column_commalist AS query_spec opt_with_check_option
`

**Comment by walter-weinmann** [comment]
Suggested solution: changing the existing grammar rules:

```
view_def -> CREATE VIEW table                                                                   : {'create view', '$3', []}.
view_def -> CREATE VIEW table '(' column_commalist ')'                                          : {'create view', '$3', '$5'}.
view_def -> AS query_spec                                                                       : {as, '$2', []                 }.
view_def -> AS query_spec WITH CHECK OPTION                                                     : {as, '$2', "with check option"}.
```

to

```
view_def -> CREATE VIEW table                                                                   : {'create view', '$3', [],   [],  []}.
view_def -> CREATE VIEW table                          AS query_spec                            : {'create view', '$3', [],   '$5, []}.
view_def -> CREATE VIEW table                          AS query_spec WITH CHECK OPTION          : {'create view', '$3', [],   '$5, "with check option"}.
view_def -> CREATE VIEW table '(' column_commalist ')'                                          : {'create view', '$3', '$5', [],  []}.
view_def -> CREATE VIEW table '(' column_commalist ')' AS query_spec                            : {'create view', '$3', '$5', '$7, []}.
view_def -> CREATE VIEW table '(' column_commalist ')' AS query_spec WITH CHECK OPTION          : {'create view', '$3', '$5', '$7, "with check option"}.

```

**Comment by walter-weinmann** [comment]
Solved with #83.
